js/src/base-component.js

```diff
@@ -17,12 +17,14 @@ const VERSION = '5.0.0-beta1'
 
 class BaseComponent {
   constructor(element) {
+    element = typeof element === 'string' ? document.querySelector(element) : element
```

Comment:
We also have https://github.com/twbs/bootstrap/blob/7e6d7b8f817ad1212238f15124335d37c93ebfea/js/src/util/index.js#L90

Not sure which solution is better/safer?

Reply:
`isElement` allows jQuery object as well :thinking:

Reply:
So, we should probably stick to `isElement`, right?

Reply:
`isElement` should not be used IMO, since Bootstrap does not allow jQuery elements in the constructors.